conference/react-js/src/components/UserMenu/UserMenu.jsx

```diff
@@ -1,28 +1,13 @@
 import React from 'react';
-import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
-import classNames from 'classnames';
 
-const menuList = data => data.map(({ title, link, last, click = null }) => (
-  <li
-    key={title}
-    className={
-      classNames(
-        'menu-list__item',
-        { 'menu-list__item_sign-out': last,
-        })}
-  >
-    <Link
-      className="menu-list__title"
-      to={link}
-      onClick={click}
-    >
-      {title}
-    </Link>
-  </li>
-));
+import MenuItem from './MenuItem';
```

Comment:
Are you sure about we need this component? We used it only once, and you can can write this easier (use li and Link with className to sign-in and out)